Source/WebCore/dom/BoundaryPoint.cpp

```diff
@@ -27,11 +27,13 @@
 #include "BoundaryPoint.h"
 #include "ContainerNode.h"
 
+#include <compare>
```

Comment:
I don’t understand where we are adding this include, where we are not adding it, and why.

Reply:
I can clean this up. At first, I thought the is_eq errors were caused by missing includes so I tried added the included in the files where it failed to build. But it didn’t help